spec/integration/apply_spec.rb

```diff
@@ -15,6 +15,7 @@
   include BoltSpec::Run
 
   let(:modulepath) { File.join(__dir__, '../fixtures/apply') }
+  let(:hiera_config) { File.join(__dir__, '../fixtures/configs/hiera.yaml') }
```

Comment:
I think this should be `../fixtures/configs/empty.yaml`. I dont see a hiera.yaml there. Which makes me wonder why it does not error?

Reply:
OH, nevermind. i see the empty file. I think you can just use `empty.yaml` so we dont have two empty files there. Seems that has been what we have been doing for empty configs other places.